src/react-navigation/navigators/createNativeBottomTabNavigator.tsx

```diff
@@ -34,8 +36,21 @@ function NativeBottomTabNavigator({
   children,
   screenListeners,
   screenOptions,
+  activeTintColor: customActiveTintColor,
+  inactiveTintColor: customInactiveTintColor,
```

Comment:
Can we match prop names from JS tabs? 

`tabBarActiveTintColor`
Color for the icon and label in the active tab.

`tabBarInactiveTintColor`
Color for the icon and label in the inactive tabs.

Also can you add those props to docs? 

https://github.com/okwasniewski/react-native-bottom-tabs/blob/main/docs/docs/docs/guides/usage-with-react-navigation.md